src/app/tests/character.service.spec.ts

```diff
@@ -40,7 +37,7 @@ describe('CharacterService', () => {
   });
 
   it('getCharacters should be defined and return the list of characters', () => {
-    const listOfCharacters: Character[] = [convertCharacter(characters[1])]
+    const listOfCharacters: Character[] = characters.map(char => convertCharacter(char));
 
     httpClientSpy.get.and.returnValue(of(listOfCharacters));
     expect(service.getCharacters()).toBeDefined();
```

Comment:
why not compare its return value to the `listOfCharacters` in some way (length, contents, or any other approach).

Reply:
that line is converting the raw json into character interface objects

Reply:
Im referring to the assertion. You're using `toBeDefined()` instead of comparing the expected output.